docs/setup/clouds/aws.md

```diff
@@ -0,0 +1,59 @@
+# Google Cloud
+
+The runner have the capability to create resources on AWS. Follow the steps below to authenticate
+and configure this service
+
+## Credentials
+
+### Get credentials for Engineering-EC2User
+
+Go on AWS Single Sign-on to get the credentials.
+Then click on the link called "Command Line or programmatic access".
+Get :
+- AWS Access Key Id
+- AWS Secret Access Key
+- AWS Session Token
+
+Create a .env file at the root of the project and put in it :
+AWS_ACCESS_KEY_ID="XXX"
+AWS_SECRET_ACCESS_KEY="XXX"
+AWS_SESSION_TOKEN="XXX"
+
+You will also have to fill in AWS_DEFAULT_REGION (us-west-2 for example) and GITHUB_TOKEN (your github token).
+
+## Config
+
+### Service
+
+Considering the application credentials has been setup on the service system,
+the remaining service configuration you'll need are:
+```yaml
+cloud_name: 'aws'
+cloud_config: {}
+```
+
+### Runner
+Here's an example of a pool of runner on aws running with ubuntu:
+```yaml
+runner_pool:
+  - config:
+      image_id: 'ami-XXX'
```

Comment:
you mentioned that the image id is public, if so, you can use it here.

Reply:
the actual value I mean. If not, as is it's perfect.

Reply:
You are right, I had wondered and in doubt I will add that this example case corresponds to Ubuntu in AWS